### ISSUE TITLE
Tile images are missing some attributes

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#421]
In `mapml-templated-tile-container` images are missing `width` and `height` attributes (the attributes are set on a parent `<div>` which has no effect):

![tile-img-no-width-height-attr](https://user-images.githubusercontent.com/26493779/115642314-03e95580-a31b-11eb-8a91-ccb618cca8ea.png)

In `mapml-static-tile-layer` images are missing `width` and `height` attributes:

![tile-img-no-role-presentation](https://user-images.githubusercontent.com/26493779/115642391-29765f00-a31b-11eb-85fe-27ab5d868438.png)
This image should also be hidden from screen readers (same as the above image), `aria-hidden="true"`  should be sufficient but for consistency with the other images it can have `role="presentation"` and empty `alt` instead (the legacy way of hiding images before `aria-hidden="true"` was broadly supported I believe). Otherwise screen readers may announce the URL of the image when the user clicks on it.
